src/bindgen/ir/enumeration.rs

```diff
@@ -700,28 +700,24 @@ impl Source for Enum {
                         for &(ref field_name, ref ty, ..) in body.fields.iter().skip(skip_fields) {
                             out.new_line();
                             match ty {
-                                Type::Array(ref _ty, ref length) => {
+                                Type::Array(ref ty, ref length) => {
                                     // arrays are not assignable in C++ so we
                                     // need to manually copy the elements
```

Comment:
I'm a bit curious why you folks use arrays like `int32_t[32]` rather than the c++ `std::array<int32_t, 32>` (which IIRC is required by spec to be standard layout & trivial, so OK to use over FFI). I'm guessing some calling conventions care about the distinction between arrays and arrays in aggregates when passing by value?

The stdlib type has the advantage of being copyable etc. which would mean that this sort of braced-initialization wouldn't be necessary.

Reply:
I don't use them :-)